benchflow-dsl/src/test/scala/cloud/benchflow/dsl/definition/configuration/goal/explorationspace/explorationvalues/IntValuesTest.scala

```diff
@@ -50,4 +50,92 @@ class IntValuesTest extends JUnitSuite {
 
   }
 
```

Comment:
We are safe from https://en.wikipedia.org/wiki/Inverse_element#Real_numbers for now because we just support integer numbers as steps.